### PR TITLE
Add detailed version-specific documentation Docker images

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

**Comment by benbc** [comment]
@jexp Please could you take a look at this. When you're happy with it I'll fork the 3.x version.

**Comment by benbc** [comment]
@jexp I don't understand the test failure here.

File: integration/docker/docker-2.x/docker-2.x.adoc [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+:level: Intermediate
+:toc:
+:toc-placement!:
+:toc-title: Contents
+:toclevels: 1
+:section: Neo4j Integrations
+:section-link: integration
+
+[NOTE]
+Docker does not run natively on Mac OSX or Windows. For running Docker on https://docs.docker.com/engine/installation/mac/[Max OSX] and https://docs.docker.com/engine/installation/windows/[Windows] please consult the Docker documentation.
```
Comment (jexp) [inline integration/docker/docker-2.x/docker-2.x.adoc]: Max OS ? x->c

**Comment by jexp** [comment]
LGTM

So if you can make the PR it against the 3.0 branch I'll merge it in.

We probably also want to add a note & link to the main /developer/docker  page to point to this page?

**Comment by benbc** [comment]
@jexp Thanks. Yes it will need to be linked to. We also need to change the various Docker Hub and Github repos to point here -- so I thought we could figure that out later.

**Comment by benbc** [comment]
@jexp I've pushed the 3.0 page as well. Please can you merge?

**Comment by jexp** [comment]
merged manually thanks a lot @benbc

**Pull request closed without merging**